akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventsourcedReplayingEvents.scala

```diff
@@ -149,6 +149,7 @@ private[persistence] class EventsourcedReplayingEvents[C, E, S](override val set
    * @param message the message that was being processed when the exception was thrown
    */
   protected def onRecoveryFailure(cause: Throwable, sequenceNr: Long, message: Option[Any]): Behavior[InternalProtocol] = {
+    setup.onRecoveryFailure(cause)
```

Comment:
surround with try catch because we want to unconditionally stop also if the callback throws

Reply:
Good catch